### DISCUSSION TITLE
Running code on Custom Dataset gives 'edge_index' error

### DISCUSSION THREAD

**Discussion by Aster77** [pyg-team/pytorch_geometric#6499] [General]
Hello,

I am trying to use Pytorch Geometric on my own custom Dataset. However I keep getting this error "Please ensure that all indices in 'edge_index' point to valid indices in the interval [0, 13) in your node feature matrix and try again." 

I checked that the max of edge_index is not greater than the number of nodes. Can anyone explain whats happening?

I have tried redoing my adjacency and edge_index matrices and I don't have this problem on InMemory datasets. Since I am making my own dataset, I see this error!

**Comment by rusty1s** [comment]
What is the output of `data.validate()`? :)
